src/templates/kit/Voting.js

```diff
@@ -0,0 +1,288 @@
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useReducer,
+  useState,
+} from 'react'
+import {
+  Field,
+  GU,
+  Help,
+  Info,
+  TextInput,
+  useTheme,
+  useViewport,
+} from '@aragon/ui'
+import { Header, PercentageField, PrevNextFooter } from '.'
+
+const MINUTE_IN_SECONDS = 60
+const HOUR_IN_SECONDS = MINUTE_IN_SECONDS * 60
+const DAY_IN_SECONDS = HOUR_IN_SECONDS * 24
+
+const DEFAULT_SUPPORT = 50
+const DEFAULT_QUORUM = 15
+const DEFAULT_DURATION = DAY_IN_SECONDS
+
+function reduceFields(fields, [field, value]) {
+  if (field === 'duration') {
+    return { ...fields, duration: value }
+  }
+  if (field === 'quorum') {
+    return {
+      ...fields,
+      quorum: value,
+      support: Math.max(fields.support, value),
+    }
+  }
+  if (field === 'support') {
+    return {
+      ...fields,
+      support: value,
+      quorum: Math.min(fields.quorum, value),
+    }
+  }
+  return fields
+}
+
+function Voting({ back, data, fields, next, screenIndex, screens }) {
+  const [{ support, quorum, duration }, updateField] = useReducer(
+    reduceFields,
+    {
+      support: data.support || DEFAULT_SUPPORT,
+      quorum: data.quorum || DEFAULT_QUORUM,
+      duration: data.duration || DEFAULT_DURATION,
+    }
+  )
+
+  const handleSupportChange = useCallback(value => {
+    updateField(['support', value])
+  }, [])
+
+  const handleQuorumChange = useCallback(value => {
+    updateField(['quorum', value])
+  }, [])
+
+  const handleDurationChange = useCallback(value => {
+    updateField(['duration', value])
+  }, [])
+
+  const handleNext = useCallback(() => {
```

Comment:
I was able to have time at zeros (0 days, 0 hours and 0 minutes) and when I go back the value used is 1 day.